warehouse/query-core/src/main/java/datawave/query/attributes/Document.java

```
@@ -855,4 +917,35 @@ public boolean isIntermediateResult() {
         return intermediateResult;
     }
     
+    private byte[] getOffsetMapToBytes() {
+        try (ByteArrayOutputStream baos = new ByteArrayOutputStream()) {
+            try (ObjectOutputStream oos = new ObjectOutputStream(baos)) {
+                oos.writeObject(offsetMap);
+                oos.flush();
+                return baos.toByteArray();
+            }
+        } catch (Exception e) {
+            log.error("something went wrong trying to serialize the offset map to a byte array");
+        }
+        return null;
+    }
+    
+    private TermOffsetMap getOffsetMapFromBytes(byte[] data) {
+        try (ByteArrayInputStream bais = new ByteArrayInputStream(data)) {
+            try (ObjectInputStream ois = new ObjectInputStream(bais)) {
+                return (TermOffsetMap) ois.readObject();
+            }
+        } catch (Exception e) {
+            log.error("failed to deserialize offset map from byte array");
```

don't swallow this exception

---

fixed in next commit